src/sql/src/session/vars.rs

```diff
@@ -925,6 +925,15 @@ pub const ENABLE_SESSION_RBAC_CHECKS: ServerVar<bool> = ServerVar {
     internal: false,
 };
 
+pub const ENABLE_SESSION_CARDINALITY_ESTIMATES: ServerVar<bool> = ServerVar {
```

Comment:
nit: maybe put a comment that in the long term this should be replaced by a `SelectOption`.